src/node/utils/Settings.js

```diff
@@ -298,6 +298,12 @@ exports.scrollWhenFocusLineIsOutOfViewport = {
  */
 exports.exposeVersion = false;
 
+/*
+ * If true, importing to a pad is allowed only if an author has a session
+ * estabilished and has already contributed to that specific pad.
+ */
+exports.requireAuthorSessionToImport = true;
```

Comment:
I like this being flipped on for future releases. But it is certainly a breaking change wrt old versions.

Are we going to keep it? If so, let's mention the change in `CHANGELOG.md`.

Reply:
I mean, IMHO this is a security patch.  I'd say it's required by default.  Also sessions here is confusing..  An author session is not the same as an Etherpad session.

Author session = Socket IO author session IE user is on pad.
Etherpad session = required for logging into "group pads"

Don't get confused!